webapp/src/main/java/com/box/l10n/mojito/android/strings/AndroidStringDocumentWriter.java

```diff
@@ -147,6 +147,6 @@ private void setAttribute(Element element, String name, String value) {
     private static String escapeQuotes(String str) {
         return Strings.nullToEmpty(str)
                 .replaceAll("\"", "\\\\\"")
-                .replaceAll("\n", "\\\\\n");
+                .replaceAll("\n", "\\\\n");
```

Comment:
that wasn't tested in previous implementation? can you add tests please?

Reply:
Tests added